libtenzir/builtins/contexts/lookup_table.cpp

```diff
@@ -182,7 +182,8 @@ class ctx final : public virtual context {
       auto column = params["field"];
       if (not column) {
         return caf::make_error(ec::invalid_argument,
-                               "missing 'field' parameter");
+                               "missing 'field' parameter for lookup "
+                               "table-based lookup");
```

Comment:
The hyphens are a bit off in this one as well. How about this?

> missing 'field' parameter for lookup in lookup-table

Reply:
Applied your suggestion.